sqlglot/expressions.py

```diff
@@ -88,18 +88,18 @@ def text(self, key):
             return field.this
         return ""
 
-    def find_comment(self, key: str) -> str:
+    def find_comments(self, key: str) -> t.List[str]:
         """
-        Finds the comment that is attached to a specified child node.
+        Finds the comments that are attached to a specified child node.
 
         Args:
             key: the key of the target child node (e.g. "this", "expression", etc).
 
         Returns:
-            The comment attached to the child node, or the empty string, if it doesn't exist.
+            The comments attached to the child node, or the empty list, if it doesn't exist.
         """
         field = self.args.get(key)
-        return field.comment if isinstance(field, Expression) else ""
+        return field.comments if isinstance(field, Expression) else []
```

Comment:
well since line 53 is always [], then you don't need it otherwise make line 53 self.comments default to None which may be better performance wise

Reply:
I think the point of this method was to avoid doing `None.comments`, which happened when we allow partial expressions. Should have probably documented this better.

Reply:
ah i misread it nvm

Reply:
`find_comments` is not actually used anywhere, so I think we can get rid of it for now.